webapp/src/Service/ImportExportService.php

```diff
@@ -668,21 +853,36 @@ protected function importTeamsTsv(array $content, string &$message = null): int
                     ->from(TeamAffiliation::class, 'a')
                     ->select('a')
                     ->andWhere('a.externalid = :externalid')
-                    ->setParameter(':externalid', $teamItem['team_affiliation']['externalid'])
+                    ->setParameter(':externalid',
+                        $teamItem['team_affiliation']['externalid'])
```

Comment:
I think this reformat and below are better left as is: both to not clutter the diff, but also because they're not changing it to PSR either.